posthog/models/error_tracking/error_tracking.py

```diff
@@ -73,6 +73,8 @@ class ErrorTrackingStackFrame(UUIDModel):
     symbol_set = models.ForeignKey("ErrorTrackingSymbolSet", on_delete=models.CASCADE, null=True)
     contents = models.JSONField(null=False, blank=False)
```

Comment:
Do we have the lineno and demangled code as part of the frame? I'm guessing yes but both of those things are pretty handy to display this nicely on the frontend